generators/client/templates/angular/webpack/webpack.dev.js.ejs

```diff
@@ -79,12 +80,26 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
         },
         {
             test: /\.ts$/,
-            loaders: [
-                'angular2-template-loader',
-                'awesome-typescript-loader',
-                'angular-router-loader'
+            use: [
+                { loader: 'angular2-template-loader' },
+                { loader: 'cache-loader' },
+                {
+                    loader: 'thread-loader',
+                    options: {
+                        // there should be 1 cpu for the fork-ts-checker-webpack-plugin
+                        workers: require('os').cpus().length - 1
+                    }
+                },
+                {
+                loader: 'ts-loader',
+                    options: {
+                        transpileOnly: true,
+                        happyPackMode: true
+                    }
+                },
+                { loader: 'angular-router-loader' }
             ],
-            exclude: ['node_modules/generator-jhipster']
+            exclude: ['node_modules']
```

Comment:
Is this giving the speed 🤔

Reply:
Nope, it doesn't seem needed, but i found a weird error in yarn start so need to check if thats from this